update-resources.sh

```diff
@@ -304,6 +305,20 @@ function add_ligolo-ng() {
   add-to-list "ligolo-ng,https://github.com/nicocha30/ligolo-ng,Advanced yet simple tunneling/pivoting tool that uses a TUN interface"
 }
 
+function add_pyexe() {
+  info "Downloading Pyexe"
+  chkfs "./windows/pyexe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py27-64.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py27.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py35-64.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py35.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py36-64.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py36.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py37-64.exe"
+  wget -O ./windows/pyexe "https://github.com/manthey/pyexe/releases/v18/download/py37.exe"
```

Comment:
You're overwriting `pyexe` at every line
Also, the links are broken, you inverted `v18` and `download`
In my opinion the files are too heavy, we shouldn't install them all. Let's keep py37-64 only
Finally, we should probably make this dynamic to avoid having to update the tag every now and then.
In conclusion, we should probably replace the whole block with the following command (and remove the `chkks` command)
```sh
wget -O ./windows/pyexe_37_64bits.exe "https://github.com/manthey/pyexe/releases/latest/py37.exe"
```

Reply:
It isn't possible to retrieve a specific asset in the latest version with a single link. However it is possible with multiple command. Is it ok for you ? 
Also there hasn't been any update on the repo in 6 years, I think that if there's an update, it will only be for new python version and python 3.7 could be remove, so it's safer to keep a pinned version.